elpy/blackutil.py

```diff
@@ -30,6 +30,12 @@ def parse_version(*arg, **kwargs):
         black = None
     else:
         import black
+        current_version = parse_version(black.__version__)
+        if current_version >= parse_version("21.5b1"):
+            from black.files import find_pyproject_toml
+        elif current_version >= parse_version("20.8b0"):
+            from black import find_pyproject_toml
+
 except ImportError:  # pragma: no cover
     black = None
```

Comment:
Also put `find_project_toml = None` as a safety measure

Reply:
actually ignore this one, if black isn't found, then the entire function won't run. Fine as is